.meta/sinks/humio.toml

```diff
@@ -0,0 +1,49 @@
+[sinks.humio]
+title = "Humio"
+batch_max_size = 1049000
+batch_timeout_secs = 1
+buffer = true
+common = false
+beta = true
+delivery_guarantee = "at_least_once"
+egress_method = "batching"
+function_category = "transmit"
+healthcheck = true
+input_types = ["log"]
+request_rate_limit_duration_secs = 1
+request_rate_limit_num =10
+request_retry_attempts = 5
+request_retry_initial_backoff_secs = 1
+request_retry_max_duration_secs = 10
+request_in_flight_limit = 10
+request_timeout_secs = 60
+write_to_description = "[Humio][urls.humio] via the [HEC API][urls.humio_hec]"
+
+[sinks.humio.options.token]
+type = "string"
+common = true
+examples = ["${TOKEN_ENV_VAR}", "A94A8FE5CCB19BA61C4C08"]
+required = true
+description = "Your Humio ingestion token."
+
+[sinks.humio.options.host]
+type = "string"
+examples = ["http://myhumiohost.com"]
+required = false
+description = "The optional host to send Humio logs to."
+
+[sinks.humio.options.encoding]
+type = "string"
+examples = ["text", "json"]
+default = "json"
+required = false
+description = """
+The encoding type that will be used to send logs. \
```

Comment:
We also support metrics.  Should we just finish off this PR for logs and then try something else for metrics? https://www.elastic.co/guide/en/beats/metricbeat/current/index.html

Reply:
Yeah, I think for now it might make sense to turn this one into `humio_logs`? It looks like if we want to support metrics we would need to instead wrap the elasticsearch sink and implement this issue https://github.com/timberio/vector/issues/422.

Since this is implemented yet, I am leaning to changing this sink to be called `humio_logs` to leave space for a `humio_metrics`.

Reply:
Agree! Especially if we're ingesting through a separate endpoint. @gburd that's cool that you support metrics. Do you basically treat them like events?